tests/plugins_test.go

```diff
@@ -44,6 +44,7 @@ exec docker run --log-driver=none --rm -i {{.ImageName}}:{{.Version}} "$@"
 	images = []string{
 		"eliza",
 		"petapis",
+		"grpc-gateway",
```

Comment:
By adding this here I think it will require plugin.sum files to be generated for this input for all existing plugins. I'd rather we only test this input when testing the gateway plugin.

Reply:
Ahh sorry, yes let me fix that up. You did already mention that, i think